code-reviews/recipes/azure-pipelines-yaml.md

```diff
@@ -0,0 +1,108 @@
+# Code Review for Azure Pipelines YAML Files
```

Comment:
See the overarching comment - this format deviates from the format that we have used for all the recipes in this section, can you re-format the content to match the other code review recipes? - E.g. YAML Code Reviews etc.

Reply:
Thanks @TessFerrandez , I've updated the document. I also added `YAML in Azure Pipelines Overview` section I think it'll be important to understand key concepts. If you can review again that would be great.